src/Controller/SsoController.php

```diff
@@ -3,13 +3,15 @@
 namespace App\Controller;
 
 use App\Entity\User;
+use Doctrine\ORM\NonUniqueResultException;
 use Doctrine\ORM\NoResultException;
+use Doctrine\Persistence\ManagerRegistry as PersistenceManagerRegistry;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Request;
 
 class SsoController extends BaseController
 {
-    public function login(Request $request)
+    public function login(Request $request, PersistenceManagerRegistry $doctrine)
```

Comment:
This doesn't work unfortunately.

You have to inject the dependency using dependency-injection.
Example can be found here:
https://stackoverflow.com/questions/70211690/new-alternative-for-getdoctrine-in-symfony-5-4-and-up